modules/carto/src/layers/vector-tile-layer.ts

```diff
@@ -151,6 +151,12 @@ export default class VectorTileLayer<
   getPickingInfo(params) {
     const info = super.getPickingInfo(params);
 
+    // MVT tiles use tile-relative coordinates, handled by MVTLayer#getPickingInfo.
+    if (this.state.mvt) {
```

Comment:
While this fixes the issue, I notice that it code path below will mean that `binaryToGeojson` is invoked twice. We could do better, either by invoking `super.super.getPickingInfo` here (https://stackoverflow.com/a/49056678) or modifying the `MVTLayer` to move `isWGS84` to state and overriding it in `VectorTileLayer`. @donmccurdy could you give this a try in a follow-up PR?

Reply:
Good call, thanks! Opened a followup PR in https://github.com/visgl/deck.gl/pull/8932.